dataland-frontend/src/components/resources/frameworkDataSearch/sfdr/SfdrDataModel.ts

```diff
@@ -365,7 +363,7 @@ export const sfdrDataModel = [
           {
             name: "nonRenewableEnergyConsumptionOtherInGWh",
             label: "Non-Renewable Energy Consumption Other",
-            description: "Energy consumption from any other available (used) non-renewable source of energy",
+            description: "Energy consumption from any other available (used) non-renewable source of energy ",
```

Comment:
was this accidentally added in the DD?

Reply:
Probably. Since it would be an overkill to open a new devtools PR, how about changing it in the DD and changing it manually in the frontend, such that next time when somebody encounters SFDR via devtools, this change will automatically be included?